scripts/datacontainer/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM alpine:3.8
+
+RUN apk update && \
+    apk add git
+
+RUN mkdir -p /usr/data/specrepo/
+
+WORKDIR /usr/data/
+RUN git clone https://github.com/sanjaiganesh/azure-rest-api-specs.git ./specrepo
```

Comment:
If you want to reduce the repository size in half, you can "git init" / "git remote add" / "git fetch" and don't checkout in Dockerfile, then checkout in run.sh.

Reply:
done